docs/articles/documentation/guides/continuous-integration/circleci-and-lambdatest.md

```diff
@@ -0,0 +1,163 @@
+
+After you pasted the YAML, click **Add Config**.
+
+![Click Add Config to save the configuration](../../../images/circleci-lambdatest/circleci-config.png)
+
+## Step 4 - Specify LambdaTest Credentials
+
+CircleCI allows you to assign sensitive information, like user credentials or access keys, to the CI project so that you do not keep secret values in the repository.
+
+Open the **Dashboard** view and select the `testcafe-ci-demo` project from the drop-down list:
+
+![Select a project from the drop-down list](../../../images/circleci-lambdatest/select-project.png)
+
+Click **Project Settings**:
+
+![Click the Project Settings button](../../../images/circleci-lambdatest/project-settings.png)
+
+Open the **Environment Variables** category and click **Add Environment Variable**:
+
+![Click the Add Environment Variable button in Project Settings](../../../images/circleci-lambdatest/add-env-variable.png)
+
+In a separate browser tab or window, open the LambdaTest app and click **Automation** in the navigation bar:
+
+![Navigate to the Automation page in the LambdaTest app](../../../images/circleci-lambdatest/lambdatest-automation.png)
+
+Click ![Key icon](../../../images/circleci-lambdatest/lambdatest-key-icon.png) to display the access key info. Copy the *username*:
+
+![Copy the username from LambdaTest Access Key Info](../../../images/circleci-lambdatest/lambdatest-credentials.png)
+
+In CircleCI, enter `LT_USERNAME` as the variable name, paste the copied value below, and click **Add Environment Variable**:
+
+![Add the LT_USERNAME environment variable in CircleCI](../../../images/circleci-lambdatest/specify-secret-variable.png)
+
+In the same manner, copy the *access key* from LambdaTest and create the `LT_ACCESS_KEY` environment variable in CircleCI to store this key.
+
+## Step 5 - Install the LambdaTest Browser Provider and JUnit XML Reporter
+
+Install the following npm packages locally in your project directory:
+
+* [testcafe-browser-provider-lambdatest](https://www.npmjs.com/package/testcafe-browser-provider-lambdatest) – the LambdaTest browser provider that enables TestCafe to run tests in the LambdaTest cloud,
+* [testcafe-reporter-xunit](https://www.npmjs.com/package/testcafe-reporter-xunit) – the JUnit XML reporter that generates test run reports in the XML format. CircleCI can process XML reports and display test results in the UI.
+
+Run the following command to install both packages:
+
+```sh
+npm install --save-dev testcafe-browser-provider-lambdatest testcafe-reporter-xunit
+```
+
+Commit the `package-lock.json` file generated by npm into the repository. This file allows CircleCI to use a fast `npm ci` command to install the dependencies.
+
+## Step 6 - Add the `test` script to package.json
+
+To specify how CircleCI should launch TestCafe, add the `test` script to the project's `package.json` file. Use the `testcafe` command in the script to run tests on LambdaTest.
```

Comment:
I've followed the instructions and `package.json` has not been created for me. Had to create one manually.

We probably need to run this beforehand:

```sh
npm init -y
```

Reply:
Yep, definitely! Created an empty `package.json` in the upstream.